basemaps/localization.md

```diff
@@ -12,31 +12,81 @@ Protomaps has several localization options for names used in text labels.
 
 <MaplibreMap/>
 
-## Default `name` value
+## Local Names
```

Comment:
@bdon what's the `Title Case` versus `Sentence case` convention elsewhere in docs that we should be following?

Reply:
No rules right now! I feel `Sentence case` is more natural, but let's just fix them when we see them.